packages/medusa/src/loaders/index.ts

```diff
@@ -91,12 +93,10 @@ async function loadMedusaV2({ directory, expressApp }) {
     next()
   })
 
-  const featureFlagRouter = featureFlagsLoader(configModule, Logger)
-
   const pgConnection = await pgConnectionLoader({ container, configModule })
 
   container.register({
-    [ContainerRegistrationKeys.MANAGER]: asValue(dataSource.manager),
+    // [ContainerRegistrationKeys.MANAGER]: asValue(dataSource.manager),
```

Comment:
Is this safe to do? cc: @adrien2p

Reply:
Yes i believe so, modules does not rely on the manager